ParsePush/build.gradle

```diff
@@ -0,0 +1,54 @@
+apply plugin: 'com.android.library'
+
+group = 'com.parse'
+version = '1.6.1'
```

Comment:
We delete `gradle.properties` and use the version number in `build.gradle`. The reason is different repos have different versions and release cycles, it does not make sense to have a unify version number.